### PR TITLE
Update VAN_WOTLK_19_25_Alliance.lua

### PR DESCRIPTION
**Pull request by tuberasa**
Fix for some of the coordinates in Stormwind

### CODE REVIEW

File: WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua [hunk 1]
```diff
@@ -27,8 +27,8 @@ A Aquatic Form|QID|5061|C|Druid|M|56.21,30.62|Z|Moonglade|N|From Dendrite Starbl
 F Rut'theran Village|ACTIVE|5061|C|Druid|M|44.15,45.22|Z|Moonglade|N|Fly back using the Darnassus Flight Master.|
 R Darnassus|ACTIVE|5061|C|Druid|M|55.86,89.45|Z|Darnassus|N|Walk into the portal back to Darnassus.|
 T Aquatic Form|QID|5061|C|Druid|M|35.36,08.39|Z|Darnassus|N|To Mathrengyl Bearwalker.|
-H Stormwind City|AVAILABLE|244|N|Hearth to Stormwind City and rejoin the guide.|C|Druid|TZ|Stormwind City|
-F Stormwind|AVAILABLE|244|N|Fly to Stormwind.|TZ|Stormwind City|
+H Stormwind City|AVAILABLE|244|N|Hearth to Stormwind City and rejoin the guide.|C|Druid|TZ|Stormwind, Elwynn|
+F Stormwind|AVAILABLE|244|N|Fly to Stormwind.|TZ|Stormwind, Elwynn|
```
Comment (Cagomei) [inline WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua:31]: make sure when you are chanting city names to do a /wp where. I am not entirely sure but I dont think Stormwind, Elwynn works, it should be throwing errors.

File: WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua [hunk 2]
```diff
@@ -45,7 +45,7 @@ T Hilary's Necklace|QID|3741|M|29.24,53.62|N|To Hilary.|Z|Redridge Mountains|
 F Sentinel Hill|ACTIVE|132|M|30.59,59.41|N|Fly to Sentinel Hill|TAXI|Sentinel Hill|Z|Redridge Mountains|
 T The Defias Brotherhood|QID|132|M|56.3,47.5|Z|Westfall|N|To Gryan Stoutmantle.|
 A The Defias Brotherhood|QID|135|M|56.3,47.5|Z|Westfall|N|From Gryan Stoutmantle.|PRE|132|
-F Stormwind|ACTIVE|135|M|56.56,52.65|Z|Westfall|N|On to Stormwind City.|TZ|Stormwind City|
+F Stormwind|ACTIVE|135|M|56.56,52.65|Z|Westfall|N|On to Stormwind City.|TZ|Stormwind, Elwynn|
```
Comment (Cagomei) [inline WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua:48]: same as above

File: WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua [hunk 3]
```diff
@@ -99,10 +99,10 @@ T The Defias Brotherhood|QID|142|M|56.33,47.52|N|To Gryan Stoutmantle.|Z|Westfal
 A The Defias Brotherhood|QID|155|M|55.68, 47.51|N|From The Defias Traitor. May have to wait for him to spawn. He takes off running as soon as you accept.|PRE|142|Z|Westfall|
 C The Defias Brotherhood|QID|155|M|42.53,71.69|N|Escort the NPC to the end of Moonbrook. Do not let him die.|Z|Westfall|
 T The Defias Brotherhood|QID|155|M|56.3,47.5|N|To Gryan Stoutmantle.|Z|Westfall|
-F Stormwind|ACTIVE|399|M|56.55,52.65|N|Fly to Stormwind.|TZ|Stormwind City|Z|Westfall|
-T Humble Beginnings|QID|399|M|49.19,30.27|Z|Stormwind City|N|To Baros Alexston.|
+F Stormwind|ACTIVE|399|M|56.55,52.65|N|Fly to Stormwind.|TZ|Stormwind, Elwynn|Z|Westfall|
```
Comment (Cagomei) [inline WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua:102]: same as above

File: WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua [hunk 4]
```diff
@@ -125,15 +125,15 @@ T Mathias and the Defias|QID|2360|M|68.4,70.2|Z|Westfall|N|To Agent Kearnen. You
 A Klaven's Tower|QID|2359|M|68.4,70.2|Z|Westfall|N|From Agent Kearnen. You will need at least 70 in Lockpicking to complete this quest.|C|Rogue|PRE|2360|
 C Klaven's Tower|QID|2359|M|70,75|Z|Westfall|QO|2|N|Pickpocket from the Malformed Defias Drone that roams around. If he doesn't have it you have to wait for him to despawn and respawn.|C|Rogue|
 C Klaven's Tower|QID|2359|M|70.4,73.93|Z|Westfall|QO|1|N|Sneak to the top of the tower, sap the boss and lockpick the box.|C|Rogue|
-F Stormwind City|QID|2359|M|56.55,52.65|Z|Westfall|N|Fly or run to Stormwind.|C|Rogue|P|Lockpicking;633;0+70;0|
+f Stormwind|QID|2359|M|56.55,52.65|Z|Westfall|N|Fly or run to Stormwind.|C|Rogue|P|Lockpicking;633;0+70;0|
```
Comment (Cagomei) [inline WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua:128]: lower case f is to pick up a flightpath, not to use the Flightpath to fly. It needs to be F

File: WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua (hunk 1 above)
Comment (tuberasa) [reply WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua:31]: "Stormwind City" doesn't work, that's why I changed it. The official name is "Stormwind, Elwynn"

File: WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua (hunk 4 above)
Comment (tuberasa) [reply WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua:128]: You are correct, been a while sinc ei made the flightpoint changes, so don't remember why. I'll check them all again to see if there are more weird things. Might've been caused by a Find-replace and me not paying good enough attention.

**Comment by tuberasa** [comment]
Fixed the F-f issue, the rest is as it should be. "Stormwind City" is not the name of a flightpoint.

File: WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua (hunk 4 above)
Comment (Cagomei) [reply WoWPro_Leveling/Wrath/Alliance/VAN_WOTLK_19_25_Alliance.lua]: no problem

**Pull request merged**